src/FsAutoComplete/FsAutoComplete.fsproj

```diff
@@ -91,6 +91,11 @@
   <Choose>
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">
       <ItemGroup>
+        <Reference Include="FSharp.Compiler.Service.MSBuild.v12">
+          <HintPath>..\..\packages\FSharp.Compiler.Service\lib\net45\FSharp.Compiler.Service.MSBuild.v12.dll</HintPath>

```

Comment:
There is no VSIX in this project, it's just a library. However, I cannot find how the NuGet package is build and published (nothing in `build.fsx`)